data/FilterList.json

```diff
@@ -17710,5 +17710,369 @@
     "name": "ROad-Kill",
     "syntaxId": 3,
     "viewUrl": "https://raw.githubusercontent.com/tcptomato/ROad-Block/master/road-kill-filters.txt"
+  },
+  {
+    "id": 1673,
+    "description": "Completely block Google and its services .",
+    "descriptionSourceUrl": "https://github.com/nickspaargaren/pihole-google",
+    "emailAddress": "poorpocketsmcnewhold@pm.me",
+    "homeUrl": "https://github.com/nickspaargaren/pihole-google",
+    "issuesUrl": "https://github.com/nickspaargaren/pihole-google/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc",
+    "licenseId": 11,
+    "name": "pihole-google (Domains)",
+    "syntaxId": 2,
+    "viewUrl": "https://github.com/nickspaargaren/pihole-google/raw/master/pihole-google.txt"
```

Comment:
Looks like you are missing a comma at the end of this line. That's why the build failed.

Reply:
Done. I´ve noticed that most of the other list didn´t added commas ad the end of thoses lines, but i´ve didn´t noticed that it was because this commas wasn´t needed, because it was directly closed by the end },. So, I´ve tought by mystake ViewUrl didn´t need one. Fixed it now normally.